app/models/placement.rb

```diff
@@ -0,0 +1,23 @@
+# == Schema Information
+#
+# Table name: placements
+#
+#  id          :bigint           not null, primary key
+#  address     :text
+#  postcode    :string
+#  school_name :string
```

Comment:
The majority of placements will be schools, but not *all* - for early years it's called a 'setting'. I wonder if there's a slightly more generic name we can use.

Reply:
name? placement_name?

Reply:
Yes I think `name` is ok.